Makefile

```diff
@@ -40,7 +40,7 @@ cmd/core-command/core-command:
 	$(GO) build $(GOFLAGS) -o $@ ./cmd/core-command
 
 cmd/export-client/export-client:
-	$(GO) build $(GOFLAGS) -o $@ ./cmd/export-client
+	$(GOCGO) build $(GOFLAGS) -o $@ ./cmd/export-client
```

Comment:
Why this change?

Reply:
Something in this change set introduced a dependency somewhere in export clients tree on ZeroMQ, which requires that CGO compilation is enabled.

Reply:
@brandonforster I am able to build this PR without the above change to the makefile. Please verify locally. If you are still having issues, let me know. We don't want to introduce CGo unnecessarily.
```
~/go/src/github.com/edgexfoundry/edgex-go $ git branch
  california

  master
* pr_558
[tconn@Borg9 14:02]
~/go/src/github.com/edgexfoundry/edgex-go $ make build
CGO_ENABLED=0 go build -ldflags "-X github.com/edgexfoundry/edgex-go.Version=0.7.0" -o cmd/config-seed/config-seed ./cmd/config-seed
CGO_ENABLED=0 go build -ldflags "-X github.com/edgexfoundry/edgex-go.Version=0.7.0" -o cmd/export-client/export-client ./cmd/export-client
CGO_ENABLED=1 go build -ldflags "-X github.com/edgexfoundry/edgex-go.Version=0.7.0" -o cmd/export-distro/export-distro ./cmd/export-distro
```